### PR TITLE
chore: Skip tests on doc updates

### PR DESCRIPTION
**Pull request by franciscojavierarceo**
# What this PR does / why we need it:
Skip tests on doc updates

# Which issue(s) this PR fixes:
https://github.com/feast-dev/feast/issues/4568

# Misc

### CODE REVIEW

**Comment by dmartinol** [comment]
the fix also includes commits that are part of other PRs. is there a way to configure devin for using different branches and separate the PRs?

**Comment by franciscojavierarceo** [comment]
> the fix also includes commits that are part of other PRs. is there a way to configure devin for using different branches and separate the PRs?

I asked it to make this PR after it did this one: https://github.com/feast-dev/feast/pull/4833

So I guess it didn't think to make a new branch. I'll merge this after I merge that PR.

**Review by dmartinol — APPROVED** [review]

**Pull request merged**
